src/installer/managed/Microsoft.NET.HostModel/Bundle/Bundler.cs

```diff
@@ -19,27 +21,38 @@ public class Bundler
     {
         readonly string HostName;
         readonly string OutputDir;
-        readonly bool EmbedPDBs;
         readonly string DepsJson;
         readonly string RuntimeConfigJson;
         readonly string RuntimeConfigDevJson;
 
-        readonly Trace trace;
+        readonly Trace Tracer;
         public readonly Manifest BundleManifest;
+        readonly TargetInfo Target;
+        readonly BundleOptions Options;
+
+        // Align embedded assemblies such that they can be efficiently mapped from the bundle.
+        public const int AssemblyAlignment = 16;
```

Comment:
Can you please expand the comment explaining why 16 is the right number?
(Previously you have 4096, so something must have changed).

Reply:
4K alignment was an attempt at trying to support R2R files on Windows without extra in-memory copies. But this alignment wasn't sufficient for that (each sections needs 4K alignment). Therefore, this attempt was dropped.

We align assemblies at 16 bytes, so that memory-mapping of its sections are cache aligned.